### PR TITLE
Obsidian redesign tweaks

### LINKED ISSUE #2989: Right-click menu not working on XP progress bar
Right-clicking the coloured bar itself in the xp tracker doesn't bring up right click menu

### LINKED ISSUE #3042: [New UI] Focused number background color is too similar to the text highlight color
This makes it difficult to see the highlighted text:

In this picture, "1,0" is selected

### LINKED ISSUE #3044: Focus plugin search bar when opening plugin panel
Automatically focusing the search box when opening the plugin configuration panel allows users to open and immediately search for plugins.

Personally when I want to change a plugin setting I use the search to find the plugin 90% of the time.

### PR DESCRIPTION
**Pull request by psikoi**
Small improvements taken from user feedback. Decided to publish the pr now to keep a organized list of the progress and reports.

**This PR has WIP in the title but is always merge-ready. It is wip as I may add new fixes as user reports come in.**

Current issues this fixes:

Fixes #2989 
Fixes #3042 
Fixes #3044

### CODE REVIEW

**Comment by psikoi** [comment]
To-do: Include xp tracker pop-out for progress bars. Will do later.

**Comment by jplsek** [comment]
Can you also look into the inconsistent text color in the settings ;)? I'm assuming the plugin configuration itself is supposed to be more white like the buttons and the plugin setting list.
ss 2018-05-19 18 15 48
ss 2018-05-19 18 16 03

**Comment by psikoi** [comment]
Ah yes, forgot about that, will do.

**Comment by jplsek** [comment]
Just noticed, should the text highlight for the notes also be orange?

**Comment by psikoi** [comment]
- Recolored the specific config items white
- Changed global selection colors for TextAreas

File: runelite-client/src/main/java/net/runelite/client/ui/components/IconTextField.java [hunk 1]
```diff
@@ -161,6 +159,12 @@ public void setEditable(boolean editable)
 		}
 	}
 
+	@Override
+	public boolean requestFocusInWindow()
+	{
+		return textField.requestFocusInWindow();
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/ui/components/IconTextField.java]: You should call super.requestFocusInWindow() here too.

Comment (Abextm) [reply runelite-client/src/main/java/net/runelite/client/ui/components/IconTextField.java]: Why?

Comment (deathbeam) [reply runelite-client/src/main/java/net/runelite/client/ui/components/IconTextField.java]: Because swing. I dont want to take bets

Comment (psikoi) [reply runelite-client/src/main/java/net/runelite/client/ui/components/IconTextField.java]: Was about to ask, the parent is a jpanel, and it works as intended. But if you feel it's better to have it, sure.

**Comment by psikoi** [comment]
- Added super.requestFocusInWindow() so that deathbeam can sleep confortably at night :P

**Pull request merged**